src/classes/TestLookUpController.cls

```diff
@@ -0,0 +1,75 @@
+@isTest
+public class TestLookUpController {
+    
+     @TestSetup
+    static void loadData(){
+        List<User> u = new List<User>();
+      	List<Invoice__c> inv = new List<Invoice__c>();
+        u = TestDataClass.insertUser();
+        inv = testDataClass.loadInvoices(); 
```

Comment:
Wiem że dla apexa TestDataClass i testDataClass to to samo ale dla czystości kodu już nie koniecznie. Ustalmy że zawsze klasy z dużej